src/main/java/org/isf/encounter/manager/EncounterBrowserManager.java

```diff
@@ -114,9 +118,22 @@ protected void validateEncounter(Encounter encounter) throws OHServiceException
 		if (encounter.getCode() == null) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.encounter.insertcode.msg")));
 		}
+
+		if (encounter.getClosedAt() != null && encounter.getPerformedAt().isAfter(encounter.getClosedAt())){
+			errors.add(new OHExceptionMessage(MessageBundle.getMessage("performedAt must be before closedAt")));
```

Comment:
I've asked you to add messages in in en and fr bundle and use their code like it's being done in the code. Why did you hard code messages ?

Reply:
@danielU2grow  you can leave the message like that but test you work after the test resolve this conversation